### PR TITLE
remove inactive reviewer from user-guide reviewers

### PR DESCRIPTION
**Pull request by Rozzii**
Is it okay for you too @dukov ?

### CODE REVIEW

File: docs/user-guide/src/OWNERS [hunk 1]
```diff
@@ -28,7 +27,8 @@ emeritus_approvers:
  - hardys
  - russellb
  - Xenwar
+ - dukov
```
Comment (furkatgofurov7) [inline docs/user-guide/src/OWNERS:30]: you have to move this to emeritus_reviewers

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

**Review by lentzi90 — COMMENTED** [review]
/lgtm

File: docs/user-guide/src/OWNERS (hunk 1 above)
Comment (Rozzii) [reply docs/user-guide/src/OWNERS]: Done , thanks !

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

File: docs/user-guide/src/OWNERS [hunk 2]
```diff
@@ -31,4 +30,5 @@ emeritus_approvers:
 
 emeritus_reviewers:
  - macaptain
- - namnx228
\ No newline at end of file
+ - namnx228
+ - dukov
```
Comment (smoshiur1237) [inline docs/user-guide/src/OWNERS:34]: Please move this line before macaptain to follow the alphabetical order

Comment (Rozzii) [reply docs/user-guide/src/OWNERS]: Done

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**